src/analysis/locally_convex/basic.lean

```diff
@@ -218,6 +218,24 @@ end
 
 variables [topological_space E] [has_continuous_smul 𝕜 E]
 
+/-- Scalar multiplication preserves neighborhoods of zero. -/
+lemma set_smul_mem_nhds_zero {c : 𝕜} {s : set E} (hs : s ∈ 𝓝 (0 : E)) (hc : c ≠ 0) :
+  c • s ∈ 𝓝 (0 : E) :=
+begin
+  rcases mem_nhds_iff.mp hs with ⟨U, hs', hU, hU'⟩,
+  refine mem_nhds_iff.mpr ⟨c • U, set.smul_set_mono hs', is_open.smul₀ hU hc, _⟩,
+  convert set.smul_mem_smul_set hU',
+  exact (smul_zero c).symm,
+end
+
+lemma set_smul_mem_nhds_zero_iff (s : set E) {c : 𝕜} (hc : c ≠ 0) :
+  c • s ∈ 𝓝 (0 : E) ↔ s ∈ 𝓝 (0 : E) :=
+begin
+  refine ⟨λ h, _, λ h, set_smul_mem_nhds_zero h hc⟩,
+  convert set_smul_mem_nhds_zero h (inv_ne_zero hc),
+  rw [smul_smul, inv_mul_cancel hc, one_smul],
+end
```

Comment:
These are true much more generally, as
```lean
/-- Scalar multiplication preserves neighborhoods. -/
lemma set_smul_mem_nhds_smul {𝕜} [group_with_zero 𝕜] [mul_action 𝕜 E]
  [has_continuous_const_smul 𝕜 E] {c : 𝕜} {s : set E} {x : E} (hs : s ∈ 𝓝 x)
  (hc : c ≠ 0) :
  c • s ∈ 𝓝 (c • x : E) :=
begin
  rcases mem_nhds_iff.mp hs with ⟨U, hs', hU, hU'⟩,
  refine mem_nhds_iff.mpr ⟨c • U, set.smul_set_mono hs', is_open.smul₀ hU hc, _⟩,
  exact set.smul_mem_smul_set hU',
end

lemma set_smul_mem_nhds_smul_iff {𝕜} [group_with_zero 𝕜] [mul_action 𝕜 E]
  [has_continuous_const_smul 𝕜 E] {c : 𝕜} {s : set E} {x : E} (hc : c ≠ 0) :
  c • s ∈ 𝓝 (c • x : E) ↔ s ∈ 𝓝 x :=
begin
  refine ⟨λ h, _, λ h, set_smul_mem_nhds_smul h hc⟩,
  rw [←inv_smul_smul₀ hc x, ←inv_smul_smul₀ hc s],
  exact set_smul_mem_nhds_smul h (inv_ne_zero hc),
end
```
Can you find a better place to put them?

Reply:
Either `topology/algebra/const_mul_action` or a new `topology/algebra/pointwise` file seems like it would be a reasonable choice.